frontend/src/components/ArtistHeader.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components/macro'
+import {Link} from "react-router-dom";
+
+export default function ArtistHeader({artist}) {
+
+    return (
+        <Link to={`/artist/${artist.artistName}`} style={{textDecoration: 'none'}}>
+            <Header>
+                <img src={artist.artistImageUrl} alt={'Could not be loaded'}/>
+                <h3>{artist.artistName}</h3>
+            </Header>
+        </Link>
+    )
+}
+
+const Header = styled.header`
```

Comment:
I would not expect a header within a link unless it was the actual header of the page, maybe you rather want to use a section or another tag?

Reply:
thx. done.